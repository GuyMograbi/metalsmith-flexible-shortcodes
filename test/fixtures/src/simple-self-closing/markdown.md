---
title: Markdown
---
This shows a simple self enclosing shortcode in markdown: [makro]