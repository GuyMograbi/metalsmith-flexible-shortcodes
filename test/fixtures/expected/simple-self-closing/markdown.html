<p>This shows a simple self enclosing shortcode in markdown: I&#39;m replaced :)</p>
